lib/irb/ext/tracer.rb

```diff
@@ -19,17 +20,6 @@ def use_tracer=(opt)
 end
 
 module IRB
-
-  # initialize tracing function
-  def IRB.initialize_tracer
-    Tracer.verbose = false
-    Tracer.add_filter {
-      |event, file, line, id, binding, *rests|
-      /^#{Regexp.quote(@CONF[:IRB_LIB_PATH])}/ !~ file and
-        File::basename(file) != "irb.rb"
-    }
```

Comment:
`Tracer.verbose` and `Tracer.add_filter` were available in v0.1.x and I could not find an equivalent in the 0.2.x API.